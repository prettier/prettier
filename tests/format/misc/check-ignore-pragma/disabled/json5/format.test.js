runFormatTest(import.meta, ["json5"], { checkIgnorePragma: false });
