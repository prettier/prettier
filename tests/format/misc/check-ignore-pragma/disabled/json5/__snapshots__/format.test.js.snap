// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`with-noformat-pragma.json5 - {"checkIgnorePragma":false} format 1`] = `
====================================options=====================================
checkIgnorePragma: false
parsers: ["json5"]
printWidth: 80
                                                                                | printWidth
=====================================input======================================
/** @noformat */

{hello:       "world"}

=====================================output=====================================
/** @noformat */

{ hello: "world" }

================================================================================
`;

exports[`with-noprettier-pragma.json5 - {"checkIgnorePragma":false} format 1`] = `
====================================options=====================================
checkIgnorePragma: false
parsers: ["json5"]
printWidth: 80
                                                                                | printWidth
=====================================input======================================
/** @noprettier */

{hello:       "world"}

=====================================output=====================================
/** @noprettier */

{ hello: "world" }

================================================================================
`;
